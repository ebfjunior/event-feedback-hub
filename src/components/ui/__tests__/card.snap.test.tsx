import React from 'react';
import { render } from '@testing-library/react';
import { Card, CardContent, CardDescription, CardHeader, CardAction } from '../card';

describe('ui/Card (snapshot)', () => {
	it('renders structure', () => {
		const { container } = render(
			<Card>
				<CardHeader>
					<CardDescription>Desc</CardDescription>
					<CardAction>Action</CardAction>
				</CardHeader>
				<CardContent>Content</CardContent>
			</Card>,
		);
		expect(container.firstChild).toMatchSnapshot();
	});
});


