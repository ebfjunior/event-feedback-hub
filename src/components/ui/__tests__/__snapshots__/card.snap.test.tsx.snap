// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ui/Card (snapshot) > renders structure 1`] = `
<div
  class="bg-card text-card-foreground flex flex-col gap-6 rounded-xl border py-6 shadow-sm"
  data-slot="card"
>
  <div
    class="@container/card-header grid auto-rows-min grid-rows-[auto_auto] items-start gap-1.5 px-6 has-data-[slot=card-action]:grid-cols-[1fr_auto] [.border-b]:pb-6"
    data-slot="card-header"
  >
    <div
      class="text-muted-foreground text-sm"
      data-slot="card-description"
    >
      Desc
    </div>
    <div
      class="col-start-2 row-span-2 row-start-1 self-start justify-self-end"
      data-slot="card-action"
    >
      Action
    </div>
  </div>
  <div
    class="px-6"
    data-slot="card-content"
  >
    Content
  </div>
</div>
`;
